warehouse/oidc/views.py

```diff
@@ -68,81 +96,119 @@ def oidc_audience(request):
     require_csrf=False,
     has_translations=True,
 )
-def mint_token_from_oidc(request):
-    def _invalid(errors):
-        request.response.status = 422
-        return {"message": "Token request failed", "errors": errors}
+@view_config(
+    route_name="oidc.mint_token",
+    require_methods=["POST"],
+    renderer="json",
+    require_csrf=False,
+    has_translations=True,
+)
```

Comment:
NB: The diff makes it a bit hard to read here, but this view now has two configured routes (each with its own `@view_config`): `oidc.github.mint_token` *and* `oidc.mint_token`.

Reply:
Ahhhh THIS is how you do this part!  This was actually something I meant to ask specifically about as I couldn't figure it out from the applicable docs.

Reply:
Yeah, this is not super well documented anywhere in Pyramid that I can tell 🙂 -- I only know this trick from seeing it used elsewhere in Warehouse.